rflx/error.py

```diff
@@ -109,7 +110,7 @@ def location(self) -> Optional[Location]:
 
     def __init__(self) -> None:
         super().__init__()
-        self.__errors: List[RecordFluxError.Entry] = []
```

Comment:
Why has `__errors` been renamed to `_errors`? I couldn't find any usage of it outside this class so there doesn't seem to be a reason to make it protected.

Reply:
It is used in `PyRFLXError` (which is a sub-class).

Reply:
Since `_errors` is no longer needed in PyRFLXError (fe521d7) it has been changed back to private.